README.md

```diff
@@ -116,9 +156,41 @@ Please check our [Contributing Guidelines](https://doc.stride3d.net/latest/en/co
 |Tests Windows Simple| <a href="https://teamcity.stride3d.net/viewType.html?buildTypeId=Engine_Tests_WindowsSimple&branch=master&guest=1"><img src="https://teamcity.stride3d.net/app/rest/builds/buildType:(id:Engine_Tests_WindowsSimple),branch:master/statusIcon"/></a>
 |Tests Windows D3D11|<a href="https://teamcity.stride3d.net/viewType.html?buildTypeId=Engine_Tests_WindowsD3D11&branch=master&guest=1"><img src="https://teamcity.stride3d.net/app/rest/builds/buildType:(id:Engine_Tests_WindowsD3D11),branch:master/statusIcon"/></a> 
 
-## Contributors ✨
+## 📖 Stride Documentation Landscape
+
+The Stride documentation is organized across different locations. Here's how it's structured:
+
+1. [Stride Game Engine](https://github.com/stride3d/stride) - The main repository for Stride, a free and open-source 2D and 3D game engine.
```

Comment:
> The main repository for Stride, a free and open-source 2D and 3D game engine.

Maybe keep it as just 

> The main repository for Stride

It sounds like a marketing spiel/AI generated

Reply:
I think, I was trying to bring **2D** word somehow somewhere because it isn't mentioned in the entire README 🤯 (so a new comer might wonder...) but it doesn't make sense to have it at the location you mentioned. I will make updates according your suggestions. We can word **2D** in another PR.

Reply:
Done.